### DISCUSSION TITLE
One Time Payments Stuck - Support Does not Respond

### DISCUSSION THREAD

**Discussion by goetzrobin** [community/community#146835] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Dear GitHub Support Team and Community,

I am writing here as a last resort to escalate a serious and unresolved issue regarding missing sponsorship payouts on my account. This situation has reached a critical point and requires immediate attention.

**The Issue:**
- In total, I have identified substantial discrepancies (multiple thousands of dollars) in the sponsorship payouts reflected in my activity feed versus what has been processed and disbursed to me.
- Today, I have received another significant one-time sponsorship that I am concerned will not be processed, adding to earlier missing payouts.

Despite multiple attempts to contact support, I have not received any acknowledgment or updates from GitHub's team. This lack of response is both frustrating and concerning, given the financial impact this has on my ability to fulfill obligations tied to these funds.

**Details Already Provided:**
- A detailed explanation of the discrepancies in payouts.
- Screenshots of my activity feed showing the one-time sponsorships.
- Exported payment data where the payouts are missing.
- My GitHub account name: goetzrobin (https://github.com/goetzrobin).

I urgently request clarification on:
- Why these payouts have not been processed.
- What steps are being taken to resolve this issue.
- A timeline for when I can expect the funds to be disbursed.

If further documentation or evidence is needed, I am prepared to provide it immediately. However, I have already supplied all relevant information, and this issue has persisted for weeks without resolution.

Given the severity and financial strain caused by these delays, I request a response within the next 24 hours. Transparency and communication are key, and I hope the GitHub team can address this promptly.

This is not just a technical issue but a matter of trust. I am counting on GitHub to stand by its open source contributors and ensure they are supported fairly.

Sincerely,
Robin Goetz

**Comment by mecodeatlas** [comment]
Hi @goetzrobin!

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by goetzrobin** [reply]
Thank you for the response.

Is there any other way to get help?
